.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
```

Comment:
Should be `pnpm`?

Reply:
`pnpm, yarn and npm` will use the same value is `npm` for `package-ecosystem` as described [here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem) @zlatanpham 

![image](https://github.com/consolelabs/websites/assets/71743905/400c9e2e-b1f6-4d4c-a271-e91953a8f4b9)